Sources/MapboxDirections/Intersection.swift

```diff
@@ -325,6 +363,11 @@ extension Intersection: Equatable {
             lhs.outletIndex == rhs.outletIndex &&
             lhs.approachLanes == rhs.approachLanes &&
             lhs.usableApproachLanes == rhs.usableApproachLanes &&
+            lhs.preferredApproachLanes == rhs.preferredApproachLanes &&
+            lhs.usableLaneIndication == rhs.usableLaneIndication &&
+            lhs.restStop == rhs.restStop &&
+            lhs.regionCode == rhs.regionCode &&
+            lhs.outletMapboxStreetsRoadClass == rhs.outletMapboxStreetsRoadClass &&
```

Comment:
It might be worth calling out in the changelog that this comparison is stricter around road classes and regions and rest stops.

Reply:
Okay sounds good!